src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -248,6 +249,28 @@ public OffenderDetailSummary getOffenderDetails(
                 .orElse(new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
+    @ApiOperation(value = "Returns the induction appointments for an offender by CRN", tags = "Contact and attendance")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 400, message = "Invalid request", response = ErrorResponse.class),
+            @ApiResponse(code = 404, message = "Offender does not exist"),
+            @ApiResponse(code = 500, message = "Unrecoverable error whilst processing request.", response = ErrorResponse.class)
+        })
+    @GetMapping(value = "/offenders/crn/{crn}/contact-summary/inductions")
+    public List<ContactSummary> getOffenderInitialAppointmentsByCrn(
+        final @ApiParam(name = "crn", value = "CRN of the offender", example = "X123456", required = true) @NotNull @PathVariable("crn") String crn,
+        final @ApiParam(name = "contactDateFrom", value = "Show contacts from this date", example = "2013-01-21") @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) @RequestParam(value = "contactDateFrom", required = false) Optional<LocalDate> contactDateFrom)
+        {
+            final var contactFilter = ContactFilter.builder()
+                .contactTypes(Optional.of(Arrays.asList("COAI","COVI","CODI")))
```

Comment:
very minor, perhaps it would be nice for these to be constants just so we can give the constants readable names so for instance you can hint what "COAI" means